src/main/java/com/virgilsecurity/pythia/client/VirgilPythiaClient.java

```diff
@@ -78,8 +83,7 @@ public VirgilPythiaClient() {
   /**
    * Create a new instance of {@link VirgilPythiaClient}.
    *
-   * @param baseUrl
-   *          the service url to fire requests to.
+   * @param baseUrl the service url to fire requests to.
    */
   public VirgilPythiaClient(String baseUrl) {
```

Comment:
Custom product and version for e3kit and other wrappers

Reply:
Done